.github/workflows/assign-reviewers.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     steps:
         
       - name: Assign reviewers
-        uses: dyladan/component-owners@main
+        uses: dyladan/component-owners@a0a1a67d6955b6efe190e9646e0ba536f882414a # tag: main
```

Comment:
Can we change to a previous commit to check if this is working also for branch names?

Reply:
```suggestion
        uses: dyladan/component-owners@cdaadffde64c918909ee081e3fe044b8910f56c2 # tag: main
```